.dir-locals.el

```diff
@@ -0,0 +1,4 @@
+;;; Directory Local Variables
```

Comment:
Is this supposed to be in here?

Reply:
It documents what formatting profile is used, and is automatically picked up for Emacs users. I'd love to use a cross-editor file instead, but I'm not aware of any (EditorConfig is very barebones, comparatively).